_posts/2017-08-08-CompetitiveCoding.md

```diff
@@ -0,0 +1,123 @@
+---
+layout: post
+title: "Competitive Coding: From Novice to Expert"
```

Comment:
> the post just clarifies what habits should be avoided by a noob in order to become essentially competitive some day...

Hence it's novice.
 
For more rigorous description see [ref: stages of skill acquisition](https://en.wikipedia.org/wiki/Four_stages_of_competence#The_four_stages_of_competence)